package.json

```diff
@@ -7,11 +7,10 @@
     "registry-mirror": "bin.js"
   },
   "scripts": {
-    "test": "mocha",
+    "test": "mocha && npm run lint",
```

Comment:
You require both tests and lint to pass on travis. So why not make that into one command and refactor two lines from `.travis.yml`?